test/e2e/predictor/test_autoscaling.py

```diff
@@ -158,6 +160,7 @@ def test_sklearn_kserve_cpu():
     kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
 
 
+@pytest.mark.fast
 def test_sklearn_kserve_raw():
```

Comment:
Maybe all raw deployment tests can be separated into its own test suite

Reply:
ok, just think we might need to install kserve without Knative for testing raw deployments, but we can do this later.